proxy/proxy.go

```diff
@@ -172,6 +173,7 @@ func (p *Proxy) Connect() error {
 		HeartBeatInterval: p.config.HeartBeatInterval,
 		IdleTimeout:       p.config.IdleTimeout,
 		Logger:            p.logger,
+		ConnectTimeout:    p.config.ConnectTimeout,
```

Comment:
So this `ConnectTimeout` already existed but it was not being populated anywhere?

Reply:
Yeah, it's used in cluster and session setup but isn't populated directly anywhere from CLI flags @joao-r-reis